Tests/Model/ModelManagerTest.php

```diff
@@ -403,6 +404,84 @@ public function testAssociationIdentifierType()
         $this->assertSame(42, $result[0]);
     }
 
+    public function getSortableInDataSourceIteratorDataProvider()
+    {
+        return array(
+            array(
+                $sortBy = null,
```

Comment:
You are right, they are not needed. Please remove them. Not sure what keys would do, but comments could be used instead.